samples/bluetooth/fast_pair/locator_tag/src/ui/platform_dk.c

```diff
@@ -139,12 +144,14 @@ static void run_led_thread_process(void)
 	static bool run_led_on = true;
 
 	while (1) {
-		if (atomic_get(&run_led_thread_state_app_running)) {
+		uint32_t sleep_time_ms = RUN_LED_BLINK_INTERVAL_MS;
```

Comment:
I would consider defining a workqueue here (actually you could even reuse system workqueue to save some memory). IMHO, using a separate thread per every monochromatic LED is an overkill.

Reply:
Also, works would be scheduled only if actually needed so nothing would be running under the hood unless necessary - it may allow OS to enter lower power states

Reply:
discussed, will use dedicated workq here with preemptive priority and work items for each led.